fink_client/consumer.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# Copyright 2019 AstroLab Software
+# Author: Abhishek Chauhan
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import io
+import os
+import json
+import confluent_kafka
+import fastavro
+import requests
+
+class AlertConsumer:
+    """
+    High level Kafka consumer to receive alerts from Fink broker
+    """
+
+    def __init__(self, topics, config, schema=None):
+        """Creates an instance of `AlertConsumer`
+
+        Parameters
+        ----------
+        topics : list of str
+            list of topics to subscribe
+        config: dict
+            Dictionary of configurations
+
+            username: str
+                username for API access
+            password: str
+                password for API access
+            group_id: str
+                group.id for Kafka consumer
```

Comment:
Fixed

Reply:
thanks - could you add an entry for `bootstrap.servers` in the docstring as well?

Reply:
done in 89ab0f7.
Actually I had the notion in mind that the user using the client need not worry about the actual address of fink servers and the fact that it can be specified was necessary for running tests.

Reply:
Thanks @cAbhi15, actually even though the user does not have to worry about it in most cases, having the information in the docstring can help in case the user wants to specify a specific IP (for whatever reasons).